### ISSUE TITLE
support everest CSI from Huawei

### ISSUE THREAD

**Issue by the-so6** [Lirt/velero-plugin-for-openstack#34]
Hi Lirt,
Thanks for your plugin for Openstack.
We are working on Open Telekom Cloud who is based with Huawei on top of Openstack. 
When we tried to make PV snapshot, the volume ID is `null` because of the CSI used.

We found a work around by adding this  : 

```go
else if pv.Spec.CSI.Driver == "cinder.csi.openstack.org" || pv.Spec.CSI.Driver == disk.csi.everest.io" {
		volumeID = pv.Spec.CSI.VolumeHandle
```  
 on `src/cinder/block_store.go` line 191.

Do you want that I proposed a PR or you think that is useless ?

**Comment by Lirt** [comment]
Hi @the-so6,

definitely we should do something about it if it's not working for you. 

I have one question and I hope I understand that correctly.

When using Openstack cloud, the storage is provisioned using Cinder component. Cinder is integrated with some storage backend in the background. Kubernetes uses Cinder CSI plugin to communicate with it and control lifecycle of volumes.

So Everest is something like a Cinder CSI plugin? If you replaced Cinder CSI plugin, would storage also work for you or Everest is only compatible in OTC? I'm asking just to understand this situation better.

I think regardless of the answer, you can create a PR to add Everest to this if/else part of the code to support this use-case.

**Comment by the-so6** [comment]
Hello,
Yes, Everest is a CSI develop by Huawei; and yes it's the only one CSI support by OTC.
Ok, I will push and you let m know if it is ok for you.

**Comment by Lirt** [comment]
Hi again, 

PR was merged and I created and built new release [v0.3.1](https://hub.docker.com/layers/lirt/velero-plugin-for-openstack/v0.3.1/images/sha256-294b69ecb8a41dd759ffd8fc9597a0688636579a5fea62a3d54d96a6603d3698?context=repo)
